src/OrbitQt/orbitmainwindow.h

```diff
@@ -132,6 +132,11 @@ class OrbitMainWindow final : public QMainWindow, public orbit_gl::MainWindowInt
   orbit_base::CanceledOr<void> DisplayStopDownloadDialog(
       const orbit_client_data::ModuleData* module) override;
 
+  void SetInspection(std::unique_ptr<CallTreeView> top_down_view,
+                     std::unique_ptr<CallTreeView> bottom_up_view) override;
+
+  void ClearInspection() override;
```

Comment:
My feeling is that these names are a little too generic in the OrbitMainWindow context. What about `SetCallTreeInspection` and `ClearCallTreeInspection` instead (or something similar)?

Reply:
Done. I may add it to also change the Samples tab, but I can rename it to SetSamplesInspection or similar then.